cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -3944,7 +3944,10 @@ TEST(TestArrowReaderAdHoc, WriteBatchedNestedNullableStringColumn) {
 }
 
 TEST(TestArrowReaderAdHoc, OldDataPageV2) {
-  // ARROW-17100
+// ARROW-17100
```

Comment:
nit: fairly sure this will be a lint error (the comment needs to be indented)

Reply:
My local formatter de-indented it originally (moved with the `#ifdef`), but seems to be ambivalent to the indentation now (will preserve either way). I changed it anyways.